test/filters.test.js

```diff
@@ -1,5 +1,11 @@
 'use strict'
 
+// `filterHttpHeaders` below indirectly (*very* indirectly) depends on there
+// being a configured agent, because "is-secret.js" reaches into `agent._conf`.
+require('..').start({
+  disableSend: true
+})
+
```

Comment:
REVIEW NOTE: This file and a few test files below implicitly depended on a configured agent because this code reaches into the agent singleton to get its config: https://github.com/elastic/apm-agent-nodejs/blob/ba570ba2af5243fa99cc1a36421cc2eb5ea930c3/lib/redact-secrets/is-secret.js#L26   A "configured agent" now means a "started agent", hence my starting an agent for these test files.